webview/src/shared/components/sectionContainer/InfoTooltip.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import cx from 'classnames'
+import { PlotsSection } from 'dvc/src/plots/webview/contract'
+import { SetupSection } from 'dvc/src/setup/webview/contract'
+import styles from './styles.module.scss'
+import { SectionDescription } from './SectionContainer'
+import Tooltip from '../tooltip/Tooltip'
+import { Icon } from '../Icon'
+import { Info, PassFilled, Error } from '../icons'
+
+export enum TooltipIconType {
+  PASSED = 'icon-pass-filled',
+  INFO = 'icon-info',
+  ERROR = 'icon-error'
+}
+
+const getIcon = (icon?: TooltipIconType) => {
+  const props = {
+    className: styles.infoIcon,
+    'data-testid': icon || TooltipIconType.INFO,
+    height: 16,
+    icon: Info,
```

Comment:
[I] Don't optionally overwrite this. Have clear logic branches to show how it is set.

Reply:
e.g 

```
const getIcon = (icon?: TooltipIconType) => {
  const defaultProps = {
    height: 16,
    width: 16
  }

  if (icon === TooltipIconType.ERROR) {
    return (
      <Icon
        {...{
          className: cx(styles.infoIcon, styles.errorIcon),
          'data-testid': icon,
          icon: Error,
          ...defaultProps
        }}
      />
    )
  }

  if (icon === TooltipIconType.PASSED) {
    return (
      <Icon
        {...{
          className: cx(styles.infoIcon, styles.completedIcon),
          'data-testid': icon,
          icon: PassFilled,
          ...defaultProps
        }}
      />
    )
  }

  return (
    <Icon
      {...{
        className: styles.infoIcon,
        'data-testid': TooltipIconType.INFO,
        icon: Info,
        ...defaultProps
      }}
    />
  )
}
```

if duplication is an issue pull out a `getProps` (or similar) helper function.

Reply:
Done!